src/Modules/Data.lua

```diff
@@ -541,6 +541,7 @@ data.uniques = { }
 for _, type in pairs(itemTypes) do
 	data.uniques[type] = LoadModule("Data/Uniques/"..type)
 end
+data.uniques['race'] = LoadModule("Data/Uniques/race")
```

Comment:
Since these are tied to special events I would suggest moving the file to `"Data/Uniques/Special/Races.lua"` and loading from there.

Reply:
Moved